ansible_builder/main.py

```diff
@@ -57,9 +57,9 @@ def build(self):
         print(MessageColors.OK + 'Writing partial Containerfile without collection requirements' + MessageColors.ENDC)
         self.containerfile.write()
         run_command(self.build_command)
-        self.containerfile.prepare_pip_steps()
         print(MessageColors.OK + 'Rewriting Containerfile to capture collection requirements' + MessageColors.ENDC)
         self.containerfile.prepare_system_steps()
+        self.containerfile.prepare_pip_steps()
```

Comment:
Now I realize my cognitive disconnect here

https://github.com/ansible/ansible-builder/pull/59/files#diff-2f525acfcc0134b3361215862821d769

I had made the same change of re-ordering these steps in that PR, but those changes got tied up in unrelated redesigns and has been hanging out for a while without getting merged into devel.

It's always a good idea in my book to prioritize merging uncontroversial small changes associated with larger feature branches 👍